packages/docs/content/4_utilities/text-transform.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Text transform
+showJumpToGuidance: true
```

Comment:
Is there a way to detect if there's a guidance section when rendering the header? That's probably pretty complicated. Just an idea

Reply:
I don't think so unless we start querying a `table of contents` property from the graphQL query and then traversing that to check

Reply:
Hmm, table of contents, you say? Won't we be wanting that for the in-page navigation in the future?

Reply:
And right now we don't have a purpose for the table of content data... However! In Scott's re-design, there is one, so maybe we could throw in some logic then?

Reply:
🥤

Reply:
Awesome!!